package.json

```diff
@@ -85,7 +85,7 @@
     "vscode:bundle:extension": "lerna run bundle:extension",
     "vscode:package": "npm run vscode:bundle:debugger && npm run vscode:bundle:extension && lerna run vscode:package --concurrency 1 && npm run reformat",
     "vsix:install": "find ./packages -name '*.vsix' -exec code --install-extension {} \\;",
-    "vscode:sha256": "lerna run vscode:sha256 --concurrency 1",
+    "vscode:sha256": "node ./scripts/generate-sha256.js >> ./SHA256",
```

Comment:
So previously this would run the vscode:sha256 script in all packages.  Since for GHA we have all the vsix files gathered in a single directory this didn't make sense.